### PR TITLE
Update PDEP8 text around open questions

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
@lithomas1 @phofl  wanted to first run this by you, before pushing to the PR.

This updates the "open questions" section.

### CODE REVIEW

File: web/pandas/pdeps/0008-inplace-methods-in-pandas.md [hunk 1]
```diff
@@ -290,12 +290,13 @@ lazy copy" to be triggered under Copy-on-Write. It is also hard to fix, adding a
...
+
+In the future, if there is demand for it, it could still be possible to add to option to raise a warning whenever this happens.
 This would be useful in an IPython shell/Jupyter Notebook setting, where the user would have the opportunity to delete
 unused references that are causing the copying to be triggered.
 
-For example,
+For example:
 
     :::ipython
     In [1]: import pandas as pd
```
Comment (jorisvandenbossche) [inline web/pandas/pdeps/0008-inplace-methods-in-pandas.md:302]: Do we want to keep this example? It's a quite long example (and the text is already long) and I am not sure if it is critical for the explanation. It essentially shows that with IPython, an inplace setitem actually triggers CoW because of IPython storing the variable when displayed. 
But that's also something we can mention in words as a typical example where you might unexpectedly get silent copies with `inplace=True`

File: web/pandas/pdeps/0008-inplace-methods-in-pandas.md [hunk 2]
```diff
@@ -334,16 +335,16 @@ was not inplace, since it is possible to go out of memory because of this.
...
+depends on the value of `inplace`, and that method chaining does not work.
+Those downsides are still relevant for the cases where we keep `inplace=True`. Therefore,
+it is also an option to have the method return the original object that was operated
+on inplace when `inplace=True`.
 
 Advantages:
 
 - It enables to use inplace operations in a method chain
 - It simplifies type annotations
-- It enables to change the default for `inplace` to True under Copy-on-Write
```
Comment (jorisvandenbossche) [inline web/pandas/pdeps/0008-inplace-methods-in-pandas.md:346]: I removed this advantage, because I am not sure this is actually true? (I don't recall our reasoning exactly)

Theoretically we _can_ change the default of course, because it doesn't violate CoW since it returns an indentical object and not a new object. But I am not sure we would actually consider doing that? It would be strange to change to not return a new object for just a few methods?

Comment (lithomas1) [reply web/pandas/pdeps/0008-inplace-methods-in-pandas.md:346]: +1 on the removal here.

File: web/pandas/pdeps/0008-inplace-methods-in-pandas.md (hunk 1 above)
Comment (lithomas1) [reply web/pandas/pdeps/0008-inplace-methods-in-pandas.md:302]: Not opposed to removing it, but it might be better to put it under a collapsible block.

File: web/pandas/pdeps/0008-inplace-methods-in-pandas.md [hunk 3]
```diff
@@ -334,16 +335,16 @@ was not inplace, since it is possible to go out of memory because of this.
 
 #### Return the calling object (`self`) also when using `inplace=True`?
 
-The downsides of keeping the `inplace=True` option for certain methods, are that the return type of those methods will
-now depend on the value of `inplace`, and that method chaining will no longer work.
-
-One way around this is to have the method return the original object that was operated on inplace when `inplace=True`.
+One of the downsides of the `inplace=True` option is that the return type of those methods
+depends on the value of `inplace`, and that method chaining does not work.
+Those downsides are still relevant for the cases where we keep `inplace=True`. Therefore,
```
Comment (lithomas1) [inline web/pandas/pdeps/0008-inplace-methods-in-pandas.md:340]: The wording in the last sentence feels a little verbose to me.

I'd reword it to "To address this, we can have those methods return the object that was operated on inplace when `inplace=True`".

File: web/pandas/pdeps/0008-inplace-methods-in-pandas.md (hunk 1 above)
Comment (jorisvandenbossche) [reply web/pandas/pdeps/0008-inplace-methods-in-pandas.md:302]: Yeah, that would be nice, but the problem is that our current "rendering" step of the web pages is rather basic, and it doesn't work with html `<details>` (the contents between the tags is then not rendered nicely).

Comment (jorisvandenbossche) [reply web/pandas/pdeps/0008-inplace-methods-in-pandas.md:302]: Will leave this as is for now, and look a bit into better rendering again.

File: web/pandas/pdeps/0008-inplace-methods-in-pandas.md (hunk 3 above)
Comment (jorisvandenbossche) [reply web/pandas/pdeps/0008-inplace-methods-in-pandas.md:340]: Thanks, used your suggestion.

**Comment by jorisvandenbossche** [comment]
OK, I incorporated this in one of the commits I pushed to https://github.com/pandas-dev/pandas/pull/51466 !
